### PR TITLE
Works and About myself sections

### PR DESCRIPTION
**Pull request by egichuhis**
Hello there, kindly review my PR and advice.

### Changes made:
- Add Works or Projects section
- Add About Me section
-Add Skills, Languages, Frameworks section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -52,11 +52,241 @@ <h2>I’am a Software Developer</h2>
...
+                    <!-- First Card -->
+                    <div class="card">
+                        <img src="./images/card1.png" alt="card-1" class="card-1-image">
+                        <div class="card-description">
+                            <h2 class="card-title">Multi-Post Stories</h2>
+                            <p class="supporting-text">
+                                A daily selection of privately personalized reads; no accounts or sign-ups required. has
+                                been the industry's standard
+                                dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+                            </p>
```
Comment (Debas-31) [inline index.html:71]: | Figma design | Your design |
|-------------------|------------------|
| Screenshot from 2022-11-05 11-34-10 | Screenshot from 2022-11-05 11-44-58 |

- [x] Great job on implementing the design for the work section :100: , but some issues to fix in the alignment of the items.


| Figma design | Your design |
|-------------------|------------------|
| Screenshot from 2022-11-05 11-47-14 | Screenshot from 2022-11-05 12-20-23 |

- [x] Kindly stick with the Figma template design you chose; make sure the button container's right margin is equivalent to the Figma template design. Fix that :+1:

File: index.html [hunk 2]
```diff
@@ -52,11 +52,241 @@ <h2>I’am a Software Developer</h2>
...
+                            <button>
+                                <span>See Project</span>
+                            </button>
+                        </div>
+                    </div>
+
+
+
+
+                </section>
```
Comment (Debas-31) [inline index.html]: - [x] KIndly fix the blank line space in your HTML file for the code to be clean, professional, and [good HTML practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

File: style.css [hunk 3]
```diff
@@ -138,4 +151,523 @@
...
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0;
+  gap: 24px;
+  width: 375px;
+  height: 3028px;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
```
Comment (Debas-31) [inline style.css]: - [Required] Kindly use the CSS technology `display: grid` for the projects and `display: flex` for each project to align.

**Review by Debas-31 — CHANGES_REQUESTED** [review]
> ## Hi @egichuhis  :wave:
# Status: Required Change :recycle:
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!




## Highlights 
- Linters setup :heavy_check_mark:
- GitHub flow :heavy_check_mark:  
- PR title and summary :heavy_check_mark:
- Commit message :heavy_check_mark:
## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (egichuhis) [reply index.html:71]: Fixed.

File: index.html (hunk 2 above)
Comment (egichuhis) [reply index.html]: Fixed.

File: style.css (hunk 3 above)
Comment (egichuhis) [reply style.css]: Fixed.

File: style.css [hunk 4]
```diff
@@ -53,10 +55,13 @@
...
+  height: 400px;
   left: 0;
   top: 92px;
   background: #1c1a19;
+  background-image: url(./images/after-socials.png);
+  background-repeat: no-repeat;
+  background-position: bottom 10px right 0;
 }
 
 .headline h1 {
```
Comment (bienvenuushindi) [inline style.css:67]: :orange_circle: **[OPTIONAL]**  Kindly remove,  in this CSS rule,  the height property or set its value to auto  to match the Figma Design.

Your Design
ba

Figma Design
aa

File: style.css [hunk 5]
```diff
@@ -138,4 +151,525 @@
...
+  gap: 24px;
+  width: 327px;
+  height: 107px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-top: 70px;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 40px;
```
Comment (bienvenuushindi) [inline style.css:187]: :orange_circle: **[OPTIONAL]** Kindly reduce this font-size value to match the Figma Design. what do you think :hugs: 
Your Design
bb

Figma Design
ab

File: style.css [hunk 6]
```diff
@@ -138,4 +151,525 @@
...
+  height: 48px;
+  background: #ff6b00;
+  flex: none;
+  order: 3;
+  flex-grow: 0;
+  border: none;
+}
+
+.see-project-btn span {
+  width: 100px;
```
Comment (bienvenuushindi) [inline style.css:334]: :orange_circle: **[OPTIONAL]** Kindly remove  this width property or set its value to `fit-content` to match the Figma Design. what do you think :hugs: 
Your Design
bc

Figma Design
ad

File: style.css [hunk 7]
```diff
@@ -138,4 +151,525 @@
...
+  height: 40px;
+  background: rgba(255, 255, 255, 0.24);
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+  justify-content: center;
+}
+
+.action-sec span {
+  width: 100px;
```
Comment (bienvenuushindi) [inline style.css:466]: :orange_circle: **[OPTIONAL]** Kindly remove this width property to match the Figma Design. what do you think :hugs: 

Your Design
bf

Figma Design
a5

File: style.css [hunk 8]
```diff
@@ -138,4 +151,525 @@
...
+  top: 4450px;
+  background: #1c1a19;
+  background-image: url(./images/Frame.png);
+  background-repeat: no-repeat;
+  background-position: bottom 130px right 0;
+  border-bottom: 1px solid #fff;
+}
+
+.about-title {
+  width: 327px;
```
Comment (bienvenuushindi) [inline style.css:571]: :orange_circle: **[OPTIONAL]** Kindly remove this width property or set its value to `fit-content` to match the Figma Design. what do you think :hugs: 

Your Design
bd
Figma Design
ae

File: style.css [hunk 9]
```diff
@@ -138,4 +151,525 @@
...
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #f4f5f7;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.get-resume-btn {
```
Comment (bienvenuushindi) [inline style.css:598]: :orange_circle: **[OPTIONAL]** Kindly set the border property to none to match the Figma Design. what do you think :hugs:  
Your Design
af

Figma Design
figma

**Review by bienvenuushindi — APPROVED** [review]
# STATUS: APPROVED 🟢 

## Hi @egichuhis   👋, 

Well done implementing the requested changes 👏
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Highlights 🎯 
:+1: No linter errors✔️

### Optional Suggestions
- [ ]  Please check the inline comments below.

**_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 4 above)
Comment (egichuhis) [reply style.css:67]: Fixed.

File: style.css (hunk 6 above)
Comment (egichuhis) [reply style.css:334]: Fixed

**Pull request merged**